Added examples for Radios in List Group

nkdas91 opened the pull request:
Closes #36643 

### [Live preview](https://deploy-preview-36644--twbs-bootstrap.netlify.app/docs/5.2/components/list-group/#checkboxes-and-radios)

nkdas91 committed:
Add examples for Radios in List Group

julien-deramond commented on site/content/docs/5.2/components/list-group.md:
```diff
@@ -269,6 +269,31 @@ Place Bootstrap's checkboxes and radios within list group items and customize as
+{{< example >}}
+<ul class="list-group">
+  <li class="list-group-item">
+    <input class="form-check-input me-1" type="radio" name="listGroupRadio" value="" aria-label="...">
```
In a set of radio buttons, one should be checked by default. You could add `checked` for the first one: `<input class="form-check-input me-1" type="radio" name="listGroupRadio" value="" aria-label="..." checked>`

julien-deramond commented on site/content/docs/5.2/components/list-group.md:
```diff
@@ -296,6 +321,31 @@ And if you want `<label>`s as the `.list-group-item` for large hit areas, you ca
 </div>
 {{< /example >}}
 
+{{< example >}}
```
IMO in order to lighten the documentation, this example is not necessary. I'd say that the previous one with checkboxes already shows how it works.

julien-deramond commented on site/content/docs/5.2/components/list-group.md:
```diff
@@ -269,6 +269,31 @@ Place Bootstrap's checkboxes and radios within list group items and customize as
 </ul>
 {{< /example >}}
 
+{{< example >}}
```
IMO maybe you could just display 3 radio buttons. With 5 checkboxes, 5 radio buttons, 5 checkboxes again + source code, it becomes long to scroll in the documentation.

julien-deramond reviewed (commented):
Thanks for this PR @nkdas91. Just few comments based on my opinion only to lighten this part of the documentation (except the one regarding the `checked` attribute).

nkdas91 committed:
Reduce the number of checkboxes and radios to 3 for consistency
Drop aria-label on inputs and use label
Use class stretched-link on labels to cover the whole list group item
Check the first radio by default
Remove radios from streched link examples

nkdas91 commented:
- Reduced the number of checkboxes and radios to 3.
- Used `<label>` and dropped `aria-label`.
There is a statement: "You can use them without labels, but please remember to include an aria-label attribute and value for accessibility."
Should we remove this?

- Checked the first radio.
- Used `.stretched-link` on `<label>`s to cover the whole list group item.
- Removed radios from the stretched link example.

Thanks!

ffoodd reviewed (approved):
Thanks ! LGTM now.

Regarding the sentence, I'm not sure. IMHO such way should be discouraged but it'd need to be tackles globally in docs and exemples, so don't bother in this PR.

@julien-deramond any thought?

julien-deramond commented on site/content/docs/5.2/components/list-group.md:
```diff
@@ -247,52 +247,53 @@ Place Bootstrap's checkboxes and radios within list group items and customize as
+You can use `.stretched-link` on `<label>`s to make the whole list group item clickable.
 
 {{< example >}}
 <div class="list-group">
```
```suggestion
<ul class="list-group">
```

Since we use `<li>`s. (source [Docs GitHub job](https://github.com/twbs/bootstrap/runs/7123619923?check_suite_focus=true))

julien-deramond commented on site/content/docs/5.2/components/list-group.md:
```diff
@@ -247,52 +247,53 @@ Place Bootstrap's checkboxes and radios within list group items and customize as
+    <input class="form-check-input me-1" type="checkbox" value="" id="thirdCheckboxStretched">
+    <label class="form-check-label stretched-link" for="thirdCheckboxStretched">Third checkbox</label>
+  </li>
 </div>
```
```suggestion
</ul>
```

julien-deramond reviewed (changes requested):
Small change to do reported by our GitHub workflow regarding the docs and after that LGTM :rocket:

julien-deramond committed:
Update site/content/docs/5.2/components/list-group.md

julien-deramond committed:
Update site/content/docs/5.2/components/list-group.md

julien-deramond commented:
> Regarding the sentence, I'm not sure. IMHO such way should be discouraged but it'd need to be tackles globally in docs and exemples, so don't bother in this PR.

Agreed. I'll try to look at it globally in the docs :ok_hand:

julien-deramond reviewed (approved):
I managed to modify the `<div class="list-group">` -> `<ul class="list-group">` directly from the interface. Thanks for the PR @nkdas91 :)
